Gaijinizer patch/Map042.txt

```diff
@@ -290,265 +317,293 @@ What!?#afrg
-But tabun...If \\N[1] is dead I will surely get
-really angry#MTLed
+But I would... probably... get quite... angry
+if... \\N[1] died...#Ron
 > END STRING
 
 > BEGIN STRING
 よく　わからない　けど……
 
 だから　\\N[1]　は　わたし　の　オス……
 > CONTEXT: Map042/events/5/pages/1/133/Dialogue/enemy2
-Well I don't know but...So \\N[1] is my male
-......#MTLed
+I don't... fully get... why but...
+
+\\N[1] is... my male...#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]　いじめ　ないか？
 
 いじめ　されたら　いつでも　言って。
 > CONTEXT: Map042/events/5/pages/1/137/Dialogue/enemy2
-\\N[1] Are you bullying? Whenever I get bullied,
-say it.#MTLed
+\\N[1]... you're not... being... picked on... are
+you...? Please tell me... if someone... is
+picking... on you...#Ron
 > END STRING
 
 > BEGIN STRING
 
 わたし　そいつ　きりきざむ　から
 > CONTEXT: Map042/events/5/pages/1/141/Dialogue/enemy2
-Because I#MTLed
+I'll slice... them up...#Ron
 > END STRING
 
 > BEGIN STRING
 お、おっかないこと真顔で言わないで！？
 
  大丈夫！！大丈夫だから！！
 > CONTEXT: Map042/events/5/pages/1/144/Dialogue/kazuya
-Oh, don't tell me not to be serious! ?
-All right!!Like I said, I'm fine! !#MTLed
+P- please don't say something so terrifying with
+such a straight face! 
+
+I'm fine! I'm absolutly fine!!#Ron
 > END STRING
 
 > BEGIN STRING
 エンプーサの舌って、魅了魔法の魔力があるんだよね？
 
 バーゲストみたいに…………
 > CONTEXT: Map042/events/5/pages/1/153/Dialogue/kazuya
-The tongue of the Empuse, Charmed There is
-magical power of magic, right?
-Like barghest ............#MTLed
+Your tongues have charm magic in them, right?
+
+Just like barghests...#Ron
 > END STRING
 
 > BEGIN STRING
 ウゥ………！！！バーゲスト！！バーゲスト！！
 
 グオォ！！
 > CONTEXT: Map042/events/5/pages/1/159/Dialogue/enemy3
-Wow .........!!! barghest!!barghest! !
-Guo! !#MTLed
+Ugh...!!! BARGHESTS!! BARGHESTS!!
+
+Gaaah!!#Ron
 > END STRING
 
 > BEGIN STRING
 な、なに！？
 
 どうしたの！エンプーサ！？
 > CONTEXT: Map042/events/5/pages/1/168/Dialogue/kazuya
-What! ? What's wrong! Empuse! ?#MTLed
+W- what?!
+
+What's wrong, Empuse?!#Ron
 > END STRING
 
 > BEGIN STRING
 フーッ！！
 
 フーッ………！！！
 > CONTEXT: Map042/events/5/pages/1/172/Dialogue/enemy3
-Foo!!Whoa .........!!!#MTLed
+Fogghh!!
+
+Fogghh...!!#Ron
 > END STRING
 
 > BEGIN STRING
 あー………。
 
 こりゃ、俺から説明した方がよさげか？
 > CONTEXT: Map042/events/5/pages/1/179/Dialogue/other2
-Ah………. Well, is it better to explain from me?#MTLed
+Ah...
+
+Better that I explain that to ya, 'kay?#Ron
 > END STRING
 
 > BEGIN STRING
 ど、どうしたんだよ一体……
 
 シェルターの時でも見たことないぞ、こんな暴れ方……。
 > CONTEXT: Map042/events/5/pages/1/183/Dialogue/kazuya
-How are you all...I've never seen it even in
-the shelter, how it's going to go wild ...#MTLed
+W- what the hell's going on?
+
+I've never seen her go nuts like this before, even
+back in the shelter...#Ron
 > END STRING
 
 > BEGIN STRING
 実はなぁ………エンプーサって、
 
  犬悪魔のバーゲストとヤバいくらい仲悪いんだよ。
 > CONTEXT: Map042/events/5/pages/1/187/Dialogue/other2
-Actually...... Empuse, I'm so weird as to
-barghest of the dog demon.#MTLed
+Yeah the thing is... Empuses and dog demons like
+barghests really don't get along.#Ron
```

Comment:
I feel like I want to replace "dog" with "canine" here. It feels like it covers a lot wider ground and seems less redundant...